PKGBUILD

```diff
@@ -9,11 +9,12 @@ pkgver=2021.01
 pkgrel=1
 _srcname=u-boot-${pkgver}
 pkgdesc="U-Boot for Orange Pi 4"
-arch=('aarch64')
+arch=('x86_64' 'aarch64')
```

Comment:
[`arch`](https://wiki.archlinux.org/index.php/PKGBUILD#arch) specifies the architectures the PKGBUILD is "intended to build and work on." The RK3399 is aarch64 only.

Reply:
this uboot version cannot be built in qemu for aarch64; there is a memory mapping error the compiler throws. (I don't have logs off-hand) | i can remove this, and further x86_64 relative flags. But there will be no way to compile it without the physical hardware. Without a updated and existing package to install, users will be left with no way to use this.